.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:prettier/recommended"],
   parser: "@typescript-eslint/parser",
   parserOptions: {
-    project: "./test-d/tsconfig.json",
+    project: "./tsconfig.eslint.json",
```

Comment:
I added `tsconfig.eslint.json` specifically so that eslint would pick up `vitest.config.js` without complaining.